src/server/controllers/image.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { GetModelVersionImagesSchema, GetReviewImagesSchema } from './../schema/image.schema';
+import { getModelVersionImages, getReviewImages } from './../services/image.service';
+
+export const getModelVersionImagesHandler = async ({
+  input: { modelVersionId },
+}: {
+  input: GetModelVersionImagesSchema;
+}) => getModelVersionImages({ modelVersionId });
+
+export const getReviewImagesHandler = async ({
+  input: { reviewId },
+}: {
+  input: GetReviewImagesSchema;
+}) => getReviewImages({ reviewId });
```

Comment:
Should we add try/catch in these controllers to handle db errors properly?

Reply:
Yea, I'll take care of this